main.nf

```diff
@@ -1236,28 +1244,36 @@ if (params.binning_map_mode == 'all'){
 
 process bowtie2 {
     tag "$assembler-$name"
+    publishDir "${params.outdir}/Assembly/${assembler}/${name}_QC", mode: params.publish_dir_mode,
+        saveAs: {filename -> filename.indexOf(".bowtie2-log") > 0 ? filename : null}
 
     input:
     set val(assembler), val(name), val(group), file(assembly), val(sampleToMap), val(sampleGroup), file(reads) from ch_bowtie2_input
 
     output:
     set val(assembler), val(name), file("${assembler}-${name}-${sampleToMap}.bam"), file("${assembler}-${name}-${sampleToMap}.bam.bai") into ch_assembly_mapping_for_metabat
+    set val(assembler), val(name), val(sampleToMap), file("*.bowtie2-log") into ch_assembly_mapping_stats
 
     when:
     !params.skip_binning
 
     script:
-    def name = "${assembler}-${name}-${sampleToMap}"
+    def NAME = "${assembler}-${name}-${sampleToMap}"
     def input = params.single_end ? "-U \"${reads}\"" :  "-1 \"${reads[0]}\" -2 \"${reads[1]}\""
         """
         bowtie2-build --threads "${task.cpus}" "${assembly}" ref
-        bowtie2 -p "${task.cpus}" -x ref $input | \
+        bowtie2 -p "${task.cpus}" -x ref $input 2>"${NAME}.bowtie2-log" | \
```

Comment:
```suggestion
        bowtie2 -p "${task.cpus}" -x ref $input 2>"${NAME}.bowtie2.log" | \
```

Reply:
This change might skew MultiQC reporting, have to test first.

Reply:
It actually works, done!